src/icu4c.mk

```diff
@@ -4,52 +4,80 @@ PKG             := icu4c
 $(PKG)_WEBSITE  := https://github.com/unicode-org/icu
 $(PKG)_DESCR    := ICU4C
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 56.1
+$(PKG)_VERSION  := 65.1
 $(PKG)_MAJOR    := $(word 1,$(subst ., ,$($(PKG)_VERSION)))
-$(PKG)_CHECKSUM := 3a64e9105c734dcf631c0b3ed60404531bce6c0f5a64bfe1a6402a4cc2314816
+$(PKG)_CHECKSUM := 53e37466b3d6d6d01ead029e3567d873a43a5d1c668ed2278e253b683136d948
+$(PKG)_GH_CONF  := unicode-org/icu/releases/latest,release-,,,-
 $(PKG)_SUBDIR   := icu
-$(PKG)_FILE     := $(PKG)-$(subst .,_,$($(PKG)_VERSION))-src.tgz
-$(PKG)_URL      := https://sourceforge.net/projects/icu/files/ICU4C/$($(PKG)_VERSION)/$($(PKG)_FILE)
-$(PKG)_DEPS     := cc
-
-define $(PKG)_UPDATE
-    $(WGET) -q -O- 'https://sourceforge.net/projects/icu/files/ICU4C/' | \
-    $(SED) -n 's,.*/projects/.*/.*/.*/\([0-9]\+[^"]*\)/".*,\1,p' | \
-    $(SORT) -Vr | \
-    head -1
+$(PKG)_URL      := $($(PKG)_WEBSITE)/releases/download/release-$(subst .,-,$($(PKG)_VERSION))/icu4c-$(subst .,_,$($(PKG)_VERSION))-src.tgz
+$(PKG)_DEPS     := cc $(BUILD)~$(PKG) pe-util
+
+$(PKG)_TARGETS       := $(BUILD) $(MXE_TARGETS)
+$(PKG)_DEPS_$(BUILD) :=
+
+define $(PKG)_BUILD_$(BUILD)
+    # cross build requires artefacts from native build tree
+    rm -rf '$(PREFIX)/$(BUILD)/$(PKG)'
+    $(INSTALL) -d '$(PREFIX)/$(BUILD)/$(PKG)'
+    cd '$(PREFIX)/$(BUILD)/$(PKG)' && '$(SOURCE_DIR)/source/configure' \
+        CC=$(BUILD_CC) \
+        CXX=$(BUILD_CXX) \
+        --enable-tests=no \
+        --enable-samples=no
+    $(MAKE) -C '$(PREFIX)/$(BUILD)/$(PKG)' -j '$(JOBS)'
 endef
 
 define $(PKG)_BUILD_COMMON
-    cd '$(1)/source' && autoreconf -fi
-    mkdir '$(1).native' && cd '$(1).native' && '$(1)/source/configure' \
-        CC=$(BUILD_CC) CXX=$(BUILD_CXX)
-    $(MAKE) -C '$(1).native' -j '$(JOBS)'
-
-    mkdir '$(1).cross' && cd '$(1).cross' && '$(1)/source/configure' \
+    rm -fv $(shell echo "$(PREFIX)/$(TARGET)"/{bin,lib}/{lib,libs,}icu'*'.{a,dll,dll.a})
```

Comment:
Didn't know this syntax was a thing. Looks nice; will it work in bash 3.22, though?

Reply:
bash on darwin is old:
```
% which bash
/bin/bash
% `which bash` --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin19)
Copyright (C) 2007 Free Software Foundation, Inc.
```
But brace expansion is nice. The use of `$(shell ...)` here defers the expansion of `*` so it can be provided as a literal for the actual command to evaluate.

Reply:
I never really thought about it before, but the pervasive use of backtick expansion in mxe is probably caused by reducing the amount of deferred expansions between `make` and the shell.

Reply:
> bash on darwin is old:

That's why I was asking but I already checked the manpage and it should work.